pkgs/servers/mastodon/default.nix

```diff
@@ -22,14 +22,6 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-Ngfs15YKLfSBOKju3BzpZFnenB370jId2G1g9Qy1y5w=";
   };
 
-  patches = [
```

Comment:
Can you remove the `fetchpatch` argument above as well?

Reply:
yup, thanks for the review :)